src/web/src/app/app.component.ts

```diff
@@ -44,6 +44,7 @@ export class AppComponent implements OnInit, OnDestroy {
 	showDropDown = false;
 	authorized: boolean;
 	private title$: Subscription;
+	public hasWspace: boolean;
```

Comment:
Перемести его пожалуйста ближе к authorized по тому что у них близкие назначения. И public он по умолчанию кажется.
```suggestion
	authorized: boolean;
	hasWspace: boolean;
	private title$: Subscription;
```

Reply:
Готово.